### PR TITLE
Port some OpenLoco helper functions

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Added a few helper functions from OpenLoco to make viewport code a bit simpler

### CODE REVIEW

File: src/openrct2/interface/Window.h [hunk 1]
```diff
@@ -128,6 +128,8 @@ struct rct_widget
     }
 };
 
+using ViewportCoordsXY = ScreenCoordsXY;
```
Comment (ZehMatt) [inline src/openrct2/interface/Window.h:131]: I think we should try to reduce the amount of coordinate types, we have already a lot of them imho.

Comment (duncanspumpkin) [reply src/openrct2/interface/Window.h:131]: Would it be clear enough that Contains and ContainsScreen and when to use either?

Comment (ZehMatt) [reply src/openrct2/interface/Window.h:131]: I think documenting the functions should provide a better way to distinguish what needs to be called.

Comment (duncanspumpkin) [reply src/openrct2/interface/Window.h:131]: Okay so I'll just add a bigger comment stating when to use each of the functions.

**Review by tupaschoal — APPROVED** [review]

File: src/openrct2/interface/Window.h [hunk 2]
```diff
@@ -147,6 +147,21 @@ struct rct_viewport
     ZoomLevel zoom;
     uint8_t var_11;
     VisibilityCache visibility;
+
+    // Use this function on coordinates that are relative to the viewport zoom i.e. a peeps x, y position after transforming from its x, y, z
```
Comment (tupaschoal) [inline src/openrct2/interface/Window.h:151]:
```suggestion
    // Use this function on coordinates that are relative to the viewport zoom i.e. a peeps x, y position after transforming
    // from its x, y, z
```

**Pull request merged**